front/app/containers/Admin/projects/project/nativeSurvey/FormResults/FormResultsQuestion/TextQuestion/Analysis/Question.tsx

```diff
@@ -31,8 +31,9 @@ const Question = ({
 }) => {
   const { formatMessage } = useIntl();
 
-  const largeSummariesEnabled = useFeatureFlag({
+  const largeSummariesAllowed = useFeatureFlag({
     name: 'large_summaries',
+    onlyCheckAllowed: true,
   });
```

Comment:
We should make this the same everywhere we use this flag, so we check allowed only. It's only about not allowing something if it's not in the pricing plan. We don't have a comprehensive implementation for disabling this feature. If we want that (not sure why, but that's a different question), we need this at the levels where you can click the link(s) to go to the analysis page rather than fiddling by disabling several buttons inside the page.